examples/tests/valid/inflight_handler_singleton.test.w

```diff
@@ -34,9 +38,27 @@ test "single instance of Foo" {
   let z = fn2.invoke("");
 
   expect.equal(x, "100");
+
+  // the simulator intentionally reuses the sandbox across invocations
+  // but we can't trust that this will always happen on the cloud
```

Comment:
Is the plan to eventually simulate the typical behavior of cloud providers where workers are occasionally evicted? It would be unfortunate to give users the false sense that they can rely on effectively stateful `cloud.Function`s.

Reply:
Eviction is a resource optimization. Ideally we should simulate the fact that when a function is not executing, it's process should be paused (`SIGSTOP`?) and resumed when another invocation needs to be performed.

Reply:
Reusing processes is also a resource optimization.

My point is that sharing memory between invocations is not something users should be relying on.

Reply:
For local testing I'm not sure if modeling eviction is that important, but if we're hosting apps using the simulator on Wing Cloud then I could see some eviction policy having merit. Sounds like a good idea for a future improvement